odoo/addons/test_lint/tests/test_i18n.py

```diff
@@ -0,0 +1,128 @@
+            ),
+            # Exclude directives starting with t-
+            (
+                """
+            <Component t-title="'Some String'" t-esc="some_variable"/>
+            """,
+                [],
+            ),
+            # Doesn't catch .translate props
+            (
+                """
+            <Component title.translate="'Some String'" t-esc="some_variable"/>
+            """,
+                [],
+            ),
+            # Include valid cases
+            (
+                """
+            <Component title="'Another String'" t-esc="another_variable"/>
+            <Component description="'Description here'" />
+            <Component title="'String with an escaped single quote ' inside'"/>
+            """,
+                [
+                    ("title=\"'Another String'\""),
+                    ("description=\"'Description here'\""),
+                    ("title=\"'String with an escaped single quote ' inside'\""),
+                ],
+            ),
+            # Exclude attributes starting with t- in between valid attributes
+            (
+                """
+            <Component title="'Valid Title'" t-esc="some_variable" t-title="'Should not be caught'" customProp="'Valid Prop'"/>
+            """,
+                [
+                    ("customProp=\"'Valid Prop'\""),
+                ],
+            ),
+            # Ensure it catches strings starting with capital letter and exclude others
+            (
+                """
+            <Component name="'singleword'" title="'SingleWord'" prop="'another String'"/>
+            """,
+                [
+                    ("title=\"'SingleWord'\""),
+                ],
+            ),
+        ]
+
+        error_count = 0
+        for i, (file_content, expected_matches) in enumerate(test_cases):
+            matches = [(m.group(3)) for m in self.PROPS_RE.finditer(file_content)]
+            if matches != expected_matches:
+                _logger.error("Test case %s failed: expected %s, got %s", i + 1, expected_matches, matches)
+                error_count += 1
+        self.assertEqual(error_count, 0)
+
+    def test_user_content_as_prop_is_translatable(self):
+        """
+        Checks if there are any props that does not use `.translate` and reports it.
+        """
+        error_count = 0
+        for file_path in self.iter_module_files("*.xml"):
+            with tools.file_open(file_path, "r") as f:
+                file_content = f.read()
+                for m in self.PROPS_RE.finditer(file_content):
+                    lineno = file_content[: m.start()].count("\n") + 1
+                    _logger.error(
+                        """The prop “%s” in file “%s” in the component node starting at line %s contains what looks like human-readable text. If the content of this prop is intended for display to the end user, it has to be translated, but props aren't by default. Add the .translate suffix to make the prop translatable.
```

Comment:
Let's keep it simple and to the point

```suggestion
                        """The prop “%s” in file “%s” in the component node starting at line %s contains what looks like human-readable text. If the content of this prop is intended for display to the end user, add the .translate suffix to make the prop translatable.
```